src/quicr_client.cpp

```diff
@@ -98,7 +98,14 @@ class QuicRTransportDelegate : public ITransport::TransportDelegate
     }
 
     messages::MessageBuffer msg_buffer{ data.value() };
-    client.handle(std::move(msg_buffer));
+
+    try {
+      client.handle(std::move(msg_buffer));
+    } catch (const std::exception& /* ex */) {
```

Comment:
Since we don't care about the exception this could just be `catch(...)`

Reply:
I was thinking that we might want to use it sooner than later. :)